mage/Magefile.go

```diff
@@ -1498,10 +1498,15 @@ func (l Lint) Yaml() error {
 type Database mg.Namespace
 
 // Retrieves the admin password for local postgres database.
+// WARNING: This function does not print the password for security reasons.
+// To retrieve the password, use:kubectl get secret orch-database-postgresql -n orch-database -o jsonpath='{.data.password}' | base64 --decode
 func (d Database) GetPassword() error {
-	pass, err := d.getPassword()
-	fmt.Println(pass)
-	return err
+	_, err := d.getPassword()
+	if err != nil {
+		return err
+	}
+	fmt.Println("kubectl get secret orch-database-postgresql -n orch-database -o jsonpath='{.data.password}' | base64 --decode")
```

Comment:
I don't see how providing the user the command to print the password is any more secure than just printing it ourselves. It may silence the code-scanning issue, but I don't feel like this is a valid issue. This command is the result of a direct user action (`mage database:GetPassword`). Asking the user to copy/paste a command is inefficient/inconvenient, when the end result is exactly the same as had we printed it ourselves.